modules/wine-deps.yml

```diff
@@ -0,0 +1,147 @@
+      - --disable-slurpd
+      - --disable-bdb
+      - --disable-hdb
+    cleanup:
+      - /bin
+      
+    sources:
+      - type: archive
+        url: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.47.tgz"
+        sha256: f54c5877865233d9ada77c60c0f69b3e0bfd8b1b55889504c650047cc305520b
+
+  - name: OpenCL
+    buildsystem: simple
+    x-multilib: false
+    build-commands:
+      - cp -av CL $FLATPAK_DEST/include/
+    sources:
+      - type: git
+        url: "https://github.com/KhronosGroup/OpenCL-Headers.git"
+        commit: c5a4bbeabb10d8ed3d1c651b93aa31737bc473dd
+
+  - name: libpcap
+    config-opts:
+      - --enable-ipv6
+    sources:
+      - type: archive
+        url: "https://www.tcpdump.org/release/libpcap-1.9.0.tar.gz"
+        sha256: 2edb88808e5913fdaa8e9c1fcaf272e19b2485338742b5074b9fe44d68f37019
+    cleanup:
+      - /bin
+
+  - name: v4l2-utils
+    config-opts:
+      - --disable-v4l-utils
+      - --disable-qv4l2
+      - --disable-qvidcap
+    sources:
+      - type: archive
+        url: "https://linuxtv.org/downloads/v4l-utils/v4l-utils-1.16.6.tar.bz2"
+        sha256: f9dac1878e3d5636eab7f56bb209fdfc66b94ee8a2aae54dcb4282fe63a678ae
+
+  - name: libgphoto2
+    sources:
+      - type: archive
+        url: "https://github.com/gphoto/libgphoto2/archive/libgphoto2-2_5_23-release.tar.gz"
+        sha256: 8de52fd997aceda895abad5d8d95a888bce24a1c739079cff64dae1da7039dde
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -ifv
+    cleanup:
+      - /bin
+      - /doc
+
+  - name: libcapi20
+    sources:
+      - type: archive
+        url: "http://deb.debian.org/debian/pool/main/libc/libcapi20-3/libcapi20-3_3.27.orig.tar.bz2"
+        md5: 439620a9221c0222c35fb3e7a3973700
+
+  - name: gsm
+    build-options:
+    make-install-args:
+      - INSTALL_ROOT=/app
+      - GSM_INSTALL_INC=/app/include
+    x-native-module:
+      build-options:
+        make-install-args:
+          - GSM_INSTALL_LIB=/app/lib
+    x-compat32-module:
+      build-options:
+        make-args:
+          - CC=ccache i686-unknown-linux-gnu-gcc
+          - CXX=ccache i686-unknown-linux-gnu-g++
+          - LD=i686-unknown-linux-gnu-gcc
+        make-install-args:
+          - CC=ccache i686-unknown-linux-gnu-gcc
+          - CXX=ccache i686-unknown-linux-gnu-g++
+          - LD=i686-unknown-linux-gnu-gcc
+          - GSM_INSTALL_LIB=/app/lib32
+    no-autogen: true
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: "http://www.quut.com/gsm/gsm-1.0.18.tar.gz"
+        sha256: 04f68087c3348bf156b78d59f4d8aff545da7f6e14f33be8f47d33f4efae2a10
+      - type: patch
+        path: ../patches/gsm/makefile.patch
+
+  - name: SPIRV-Headers
+    x-multilib: false
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: "https://github.com/KhronosGroup/SPIRV-Headers.git"
+        tag: 1.5.1.corrected
+
+  - name: vkd3d
```

Comment:
Should we bundle vkd3d? Seems like it's not universal, vanilla Wine, Lutris wine and Proton use different versions, so probably vkd3d should be bundled alongside wine instead.